pkg/qdr/amqp_mgmt_mock_test.go

```diff
@@ -0,0 +1,159 @@
+package qdr
+
+import (
+	"flag"
+	"strings"
+	"testing"
+
+	"crypto/tls"
+	"gotest.tools/assert"
+)
+
+var clusterRun = flag.Bool("use-cluster", false, "run tests against a configured cluster")
+
+func TestQDR(t *testing.T) {
+
+	// AsInt -----------------------------------
+	goodAsInt := Record{}
+	goodAsInt["int8"] = int8(0)
+	goodAsInt["int16"] = int16(0)
+	goodAsInt["int32"] = int32(0)
+	goodAsInt["int64"] = int64(0)
+
+	goodAsInt["uint8"] = uint8(0)
+	goodAsInt["uint16"] = uint16(0)
+	goodAsInt["uint32"] = uint32(0)
+	goodAsInt["uint64"] = uint64(0)
+
+	for key, value := range goodAsInt {
+		_, ok := AsInt(value)
+		assert.Assert(t, ok, "AsInt failed on type |%s|.", key)
```

Comment:
you could assert also that the returned value is what you expect, (cero in this case) and or, some other value... since 0  is usually the default

Reply:
Very good point about the use of bool rather than (more informative) error.

These fns are not used in very many places -- I guess I would vote to change them right now to use more golang-standard behavior.   (Which would also require changes to my tests.)

I will go ahead and make that change and then we can see if we like it.